src/test/util/event_buffer.rs

```diff
@@ -350,31 +336,22 @@ impl<'a, T: Clone> EventSubscriber<'a, T> {
     }
 
     #[cfg(feature = "in-use-encryption-unstable")]
-    pub(crate) async fn collect_events_map<F, R>(
-        &mut self,
-        timeout: Duration,
-        mut filter: F,
-    ) -> Vec<R>
+    pub(crate) async fn collect_map<F, R>(&mut self, timeout: Duration, mut filter: F) -> Vec<R>
     where
         F: FnMut(T) -> Option<R>,
     {
         let mut events = Vec::new();
         let _ = runtime::timeout(timeout, async {
-            while let Some(event) = self.filter_map_event(timeout, &mut filter).await {
+            while let Some(event) = self.next_map(timeout, &mut filter).await {
                 events.push(event);
             }
         })
         .await;
         events
     }
 
-    #[cfg(feature = "in-use-encryption-unstable")]
-    pub(crate) async fn clear_events(&mut self, timeout: Duration) {
```

Comment:
This was a now nearly nonsensical holdover from previous iterations of event cleanup, only used in a few places in CSFLE tests that were easily replaced with just making a fresh `EventStream`.